apps/devportal/data/markdown/pages/learn/getting-started/xm-cloud/tutorials/setup-xm-cloud.md

```diff
@@ -0,0 +1,182 @@
+
+Now, let’s start the creation process of the site.  
+
+5. Click `create website`, which will scaffold the website in the background. This will take a few minutes.  
+
+Once your site is scaffolded in the background, you can access it using XM Cloud Pages and start editing the content. 
+
+![Open Pages Editor](https://sitecorecontenthub.stylelabs.cloud/api/public/content/6438c1971da24b8f8564e3e48335d985?v=c5f57f17) 
+ 
+6. Click on the website, which opens up a WSYWIG editor where you can begin to edit the content, see components that you can use to edit the site, and switch between languages and your sites (top nav bar).   
+![In Pages Editor](https://sitecorecontenthub.stylelabs.cloud/api/public/content/9fb983d0eeec435797a1115f4ae801df?v=2cce57d4) 
+ 
+
+## Working locally  
+
+Let’s now navigate to your GitHub account and download the repository that was cloned earlier by the Sitecore XM Cloud Deploy App.  
+![GitHub - Clone Repository](https://sitecorecontenthub.stylelabs.cloud/api/public/content/239fa0b1ce314ca088bc830ee7a699e8?v=ba0d1b2e) 
+
+### Download Repository  
+
+1. In your repositories overview, find the newly created xm-cloud-tutorial-series repo.  
+2. Download the repository to your local file system using the GitHub CLI.Now, open the newly created folder in VS Code, or your text editor of choice.  
+3. As per the README, run the init script that was delivered with your source code repository passing the path to your XM Cloud license file, and the admin password of your XM Cloud instance (“b”, in this case). The init script initializes the .env file with everything necessary.  
+![In Command Line Tool - Run init script](https://sitecorecontenthub.stylelabs.cloud/api/public/content/8cb688885bc74cf9a3b44468dd58ff57?v=0ca77d8f) 
+ 
+ 
+*A note on passwords:  
+You can use any password and will of course have to use a secure password when dealing with client projects.  But for now we use “b”, for tradition’s sake. * 
+
+### Connect the App to the XM Cloud Instance  
+
+A few things need to be configured upfront to connect to the GraphQL endpoint.  
+
+1. Navigate to the .env file within your solution under src\sxastarter, and copy it.  
+
+2. Name it .env.local. We do this so that the file is ignored by git by default, so that you do not pass valuable configuration secrets into any repositories.  
+ 
+You can connect using the delivery endpoint of Edge or to the preview endpoint of your CM instance.   
+![XM Cloud Architecture - Connect to Preview Endpoint](https://sitecorecontenthub.stylelabs.cloud/api/public/content/e703be66170a49268beb6564c4c34df7?v=5184aace) 
+
+Any app showing content to the public must connect to the Edge Delivery Endpoint that is scaled and geographically distributed. Only published content will be available here.  For development purposes, you will connect to the preview endpoint as you do not need to care about publishing at this point, so created content is instantly available.    
```

Comment:
I keep getting lost here (and I imagine others might too) with the endpoints being different for public vs. dev content.  Can we link to an article that explains that in further depth?

Reply:
This does not yet seem to be resolved. I agree we should link out to something about endpoints.

Reply:
I added a link to Edge architecture to this step and also in the related docs at the bottom.